tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +49,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="nwork", default=3, type="int",
```

Comment:
`nwork` is an abbreviation which might not be obvious when reading the code in the future. Please use something more verbose like `num_workers` instead (throughout the patch).